### DISCUSSION TITLE
Add detection for more packaging codes

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/robotoff#1213] [Ideas]
#### Gluten codes
```GLUTEN-FREE\nGB-077-286```
Add detection for Gluten codes such as the one in this image
https://static.openfoodfacts.org/images/products/408/860/001/8089/1.jpg

#### PEFC codes
PEFC/07-32-227
https://static.openfoodfacts.org/images/products/322/888/101/0971/18.jpg
``` \nPEFC/07-32-227\n ``` 

#### Spanish codes
- #233 

### Part of
- #374

**Comment by raphael0202** [comment]
Can you give me a bit more context about what are these gluten code and how to detect them? What are they used for?

**Comment by alexgarel** [comment]
> Can you give me a bit more context about what are these gluten code and how to detect them? What are they used for?

I think they are certifications. (search gluten free certification)

First one seems to be linked to https://www.aoecs.org/
